Slim/Plugin/MusicMagic/Plugin.pm

```diff
@@ -876,7 +876,7 @@ sub getMix {
 
 	my $mixArgs = join('&', map {
 		my $id = !main::ISWINDOWS && ($validMixTypes{$for} eq 'song' || $validMixTypes{$for} eq 'album') ? Slim::Utils::Unicode::utf8decode_locale($_) : $_;
-		$validMixTypes{$for} . '=' . Plugins::MIPMixer::Common::escape($id);
+		$validMixTypes{$for} . '=' . Plugins::MusicMagic::Common::escape($id);
```

Comment:
This time we should try to get it right. I failed the review the first time...

That should be `Slim::Plugin::MusicMagic::Common::escape()`. Are you doing this without being able to actually test it?...

Reply:
Never mind, I'll just do it on my end. Thanks for the heads up!